### DISCUSSION TITLE
I want to know the rhythm, timing and related tuning options

### DISCUSSION THREAD

**Discussion by cod3r0k** [idiap/coqui-ai-TTS#124] [General]
Hi, how can I learn about how the system manages or edits rhythm, timing, and related tuning options?

**Comment by eginhard** [comment]
Coqui has implementations for many different models which handle these in different ways, but to some degree these are still open research problems as well.

**Comment by cod3r0k** [comment]
Thanks @eginhard  But I don’t know if you've tested the 'Advanced Voice Mode' of ChatGPT. From what I can tell, it works very well, and most issues seem to be resolved. I’m looking to have a similar TTS model.
